Kong/docs.konghq.com
# Update: Konnect plugin pricing tiers

### Description

Removing all instances of `paid` and `premium` plugin badges and categorization.
Adding a "konnect compatible" badge instead.

https://konghq.atlassian.net/browse/DOCU-4018

### Testing instructions
https://deploy-preview-7896--kongdocs.netlify.app/hub/

To review, check:
* [Plugin hub filters](https://deploy-preview-7896--kongdocs.netlify.app/hub/?compatibility=konnect)
* Badges on plugins - click into plugins as well
* [Konnect compatibility reference page](https://deploy-preview-7896--kongdocs.netlify.app/konnect/compatibility/#plugin-compatibility)
* [Plugin license tiers page](https://deploy-preview-7896--kongdocs.netlify.app/hub/plugins/license-tiers/)

### Checklist
